src/components/TodoForm.js

```diff
@@ -7,7 +7,7 @@ function TodoForm(props) {
 
     useEffect(() => {
         inputRef.current.focus();
-    })
+    },[])
```

Comment:
add `[]` would have stop the re-rendering. Now, this would keep on re-rendering on change of any value in the component